client/src/main/java/io/pravega/client/segment/impl/ConditionalOutputStreamImpl.java

```diff
@@ -70,7 +70,7 @@ public boolean write(ByteBuffer data, long expectedOffset) throws SegmentSealedE
                         if (client == null || client.isClosed()) {
                             client = new RawClient(controller, connectionFactory, segmentId);
                             long requestId = client.getFlow().getNextSequenceNumber();
-                            log.debug("Setting up append on segment: {}", segmentId);
+                            log.debug("Setting up appends on segment: {} for ConditionalOutputStream.", segmentId);
```

Comment:
nit: remove `:` as this part is not at the end of the message anymore.

Reply:
fixed it.